config/global.php

```diff
@@ -79,6 +79,8 @@
     'Piwik\Translation\Loader\LoaderInterface' => DI\object('Piwik\Translation\Loader\LoaderCache')
         ->constructor(DI\get('Piwik\Translation\Loader\JsonFileLoader')),
 
+    'DeviceDetector\Cache\Cache' => DI\object('Piwik\DeviceDetector\DeviceDetectorCache')->constructor(86400),
```

Comment:
Would this also work in the DeviceDetector's plugn DI config? Seems like a more appropriate place to put it to me.

Reply:
I reckon it would cause issues with the API plugin... if DeviceDetection plugin was disabled... Probably there be other places where it would cause issues too but in the end maybe not too important?

Reply:
Wouldn't that be an issue anyway if the the plugin was disabled? Or I guess the class is loaded no matter what? Something about that seems odd, but I guess it's not that important.

Reply:
Yes the class would still be loaded correctly...

Reply:
ok.